tests/xor_lookups/tests.py

```diff
@@ -18,6 +18,16 @@ def test_filter(self):
             Number.objects.filter(Q(num__lte=7) ^ Q(num__gte=3)),
             self.numbers[:3] + self.numbers[8:],
         )
+        self.assertCountEqual(
+            Number.objects.filter(
+                Q(num__gte=1)
+                ^ Q(num__gte=3)
+                ^ Q(num__gte=5)
+                ^ Q(num__gte=7)
+                ^ Q(num__gte=9)
+            ),
+            self.numbers[1:3] + self.numbers[5:7] + self.numbers[9:],
```

Comment:
For the sake of explicitness, would you add a comment before the assertion showing to the reader the actual expected list? Something like:

```suggestion
            # [1, 2, 5, 6, 9]
            self.numbers[1:3] + self.numbers[5:7] + self.numbers[9:],
```